documentation/docs/validator_recovery.md

```diff
@@ -0,0 +1,88 @@
+### Overview
+
+This document describes the steps necessary to ensure that a validator node can be restored in case its state is lots. In order to achieve this, it is necessary that the following data is safely backed up:
+
+* Tendermint validator key
+* Axelar validator mnemonic
+* Axelar proxy mnemonic
+* Tofnd mnemonic
+
+Besides the data described above, it will also be necessary to retrieve the *recovery data* associated to all the key shares that the validator was responsible for maintaining.
+
+### Tendermint validator key
+
+The Tendermint validator key is created when a node is launched for the first time.
+It can be found within the node's container at `/root/.axelar/config/priv_validator_key.json` (or from the node's root directory at `$ROOT_DIRECTORY/.core/config/priv_validator_key.json`).
```

Comment:
I assume one of them is meant to be "host" and not node?

Reply:
nop, I actually mean the node's directory that can be specified in the joinTestnet script (that we do refer as root even in that context)